metis-pattern-analysis/pom.xml

```diff
@@ -36,7 +37,6 @@
           <version>${version.apache.commons.text}</version>
```

Comment:
Where do you see that? I cannot find it.

Reply:
I see.
Well since it is in problem-patterns and it does not use zoho I'll leave it as it is.
Generally speaking the zoho libs has caused me some issues in the past so I would be hesitant to change the lib version there unless we tested it well.

Reply:
ok :+1: